scripts/globals/dark_ixion.lua

```diff
@@ -0,0 +1,471 @@
+-----------------------------------
+-- Area: One of many zones in Shadowreign zones
+--   NM: Dark Ixion
+-- https://ffxiclopedia.fandom.com/wiki/Dark_Ixion
+--[[
+    Outline of the mob's behavior:
+
+    Will run away, most of the time to another zone, if:
+    - Campaign Battle begins
+    - a player scares it by getting too close
+    - attacks or attempts to claim with a Stygian Ash but misses
+    - or if whoever has successfully claimed it wipes.
+    - His HP is preserved after wiping and him running away to another zone
+    - If Dark Ixion is due to spawn or is already spawned during maintenance, he will spawn shortly after server comes back online.
+    - If he was not due to spawn during this time frame, his spawn window will reset to 21 hours after servers come online.
+
+    Roaming: runs very fast and can be spooked via normal aggro/claim actions. If spooked, it does the same thing as attempting a claim but missing with a Stygian Ash
+    Claim: can only be claimed by landing a hit with Stygian Ash
+    Targetting: Can damage and kill players riding Chocobo with Area of Effect attacks even if player is not in the Alliance fighting him.
+
+    Mobskills:
+    - has a normal set of TP moves, but telegraphs most/all of them
+    - if he has an aura then his TP moves are doubled up back to back
+    - has a special move that isn't listed in the combat log, that many have called 'Trample'
+        - Trample: Charges forward, dealing high damage to,(400-1000) and lowering the MP (10-30%) of, anyone in his path. No message is displayed in the chat log.
+        - When Dark Ixion's HP is low, he can do up to 3 Tramples in succession.
+        - Can be avoided easily by moving out of its path.
+        - May charge in the opposite, or an entirely random, direction from the one he is currently facing.
+        - Will load a set number of targets in his path before ramming forward.
+        - Occasionally, a person in his path will not be hit, as well as those wandering in its path after it has begun its charge.
+--]]
+-----------------------------------
+require('scripts/globals/npc_util')
+
+xi = xi or {}
+xi.darkixion = xi.darkixion or {}
+
+-- TODOs, notes, and reminders
+
+-- mob:AnimationSub() -- 0 is normal || Charging is animation sub 1  || 2 is broken horn || 3 is glowing and causes horn to repair
+-- TODO: dmg taken from front/rear (if we can)
+
+xi.darkixion.zoneinfo =
+{
+    [xi.zone.JUGNER_FOREST_S] =
+    {
+        mobID = 17113468,
```

Comment:
No need to hard-code the ID here, you can do a dynamic lookup using the zone object whenever you need to mobID (everything is cached and fast):

```
    local darkIxion = zone:queryEntitiesByName('Dark_Ixion')[1]
    local mobID     = darkIxion:getID()
```

Reply:
oh, wow. OK cool I'll update things. Makes my table really just a skeleton to contain the patrol paths and valid zones.

Reply:
Would it be ok to leave the mobIDs in there as reference. Debugging is much simpler when you can `!gotoid <mobid>` unless there's a similar gm command?

Reply:
I've never used it, but it looks like this exists:

`!gotoname Dark_Ixion`